docs/testing.md

```diff
@@ -0,0 +1,103 @@
+# Testing Taps & Targets
```

Comment:
Should this mention adding a `conftest.py` with

```python
pytest_plugins = ("singer_sdk.testing.pytest_plugin",)
```

?

Reply:
Done ✅ I also added an example of `SuiteConfig` to the docs 🙂